model/node_labels.go

```diff
@@ -0,0 +1,28 @@
+package model
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+type NodeLabels map[string]string
+
+func (l NodeLabels) Enabled() bool {
+	return len(l) > 0
+}
+
+// Returns key=value pairs separated by ',' to be passed to kubelet's `--node-labels` flag
+func (l NodeLabels) String() string {
+	labels := []string{}
+	keys := []string{}
+	for k, _ := range l {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, k := range keys {
+		v := l[k]
```

Comment:
no it isn't, but seeing args on a command line with empty `=` is a little confusing, at least for me. end result is the same